src/Interactive/Host/Interactive/Core/InteractiveHost.Service.cs

```diff
@@ -140,8 +140,6 @@ private Service(Func<Func<object>, object> invokeOnMainThread)
 
                 _lastTask = Task.FromResult(initialState);
 
-                Console.OutputEncoding = Encoding.UTF8;
```

Comment:
Any idea why is this suddenly a problem? It's always worked fine.

Reply:
Exception details now added above

Reply:
Would you prefer this, or a try/catch around it?

Reply:
Testis failing, so this is actually not unnecessary. Let's add try-catch but file an issue to investigate why is this failing.

Reply:
➡️ Updated the code to catch `IOException` and report NFW without crashing